test/alignment-D3.js

```diff
@@ -142,6 +146,41 @@ describe('D3 Alignment module tests', function() {
   </circle>
   `);
 
+  const barChartDom = new JSDOM(`
```

Comment:
NOTE: I didn't spend much time reviewing this because this code should be changing soon with your improvements. I'll look closer at that PR.